src/main/java/edu/uc/inventorymanager/dao/ItemDAOStub.java

```diff
@@ -27,9 +27,9 @@ public Item save(Item item) throws Exception {
     @Override
     public Item fetchById(int id) {
         return allItems.stream()
-                .filter(item -> item.getId() == id)
-                .findAny()
-                .orElse(null);
+            .filter(item -> item.getId() == id)
```

Comment:
I realized later that these were probably deep indents, so this is probably not a good recommendation after all. However, if they are not supposed to be deep indents, they should be indented using 4 spaces instead of 8 like to be consistent with the rest of the code. Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3